src/main/java/core/basesyntax/service/impl/ValidationServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+
+public class ValidationServiceImpl implements Predicate<String> {
+    private String pattern;
+
+    public ValidationServiceImpl() {
+        pattern = "[" + getSupportedOperations() + "],\\w+,\\d+";
```

Comment:
you can also use constant with %s
and later use String.format when you want to use this constant